artemis/src/main/java/tech/pegasys/artemis/cli/deposit/DepositGenerateCommand.java

```diff
@@ -274,4 +276,20 @@ public String getPasswordEnvironmentVariable() {
       return withdrawalPasswordEnv;
     }
   }
+
+  private static class NumberOfValidatorConverter implements ITypeConverter<Integer> {
```

Comment:
Seems slightly weird that this is such a specific converter and not just a `PositiveIntegerConverter`.

Reply:
:) `PositiveIntegerConverter` has a nice ring to it, let me change to that.